common/test/acceptance/pages/common/utils.py

```diff
@@ -56,10 +56,6 @@ def confirm_prompt(page, cancel=False, require_notification=None):
     cancel is True.
     """
     page.wait_for_element_visibility('.prompt', 'Prompt is visible')
-    page.wait_for_element_visibility(
```

Comment:
She did some work to get focus-ability specifically added to bok-choy, so I'm guessing she had good reasons for adding it in. That said, I don't know what her reasons were for adding it in here, and it was clearly added before it was put into bok-choy anyway. Since we can't really ask her for her reasons right now, removing this seems fine to me.